gammapy/image/lists.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from numpy.testing import assert_allclose
+from astropy.units import Quantity
+from astropy.wcs import WCS
+from ..utils.testing import requires_dependency, requires_data
+from . import catalog
```

Comment:
Some of these imports are unused, no?
E.g. these can be removed:
```
from . import catalog
from ..irf import EnergyDependentTablePSF
```
If you edit in `PyCharm` this will be pointed out to you directly.
Alternatively, you can run e.g. https://pypi.python.org/pypi/pyflakes on your files from the command line, and it will tell you which imports are unused.